teuthology/task/install/rpm.py

```diff
@@ -1,9 +1,12 @@
 import logging
 import os.path
+import time
```

Comment:
if using contextutil.safe_while() you don't need this import

Reply:
This is what the flake8 is saying:
```
teuthology/task/install/rpm.py:3:1: F401 'time' imported but unused
teuthology/task/install/rpm.py:156:20: F821 undefined name 'CommandFailedError'
```

Reply:
Yeah, forgot to remove it. Will fix it.
Thanks.